pages/pipelines/branch_configuration.md.erb

```diff
@@ -58,12 +56,12 @@ When you define a value in the `branch` attribute of a step, only builds on the
 
 The following are examples of patterns, and the branches that they will match:
 
-* `master production` will match master and production only
-* `master features/*` will match master and any branch that starts with `features/`
-* `*/release-123` will match any branch that ends with `/release-123` such as `feature-123/release-123`
-* `v*.0` will match branch that begins with a `v` and ends with a `.0`, such as `v1.0`
-* `v*.*.*` will match branch that begins with a `v` and has two periods, such as `v1.2.3`
+* `master` will match `master` only
 * `!production` will match any branch that's not `production`
-* `staging !production development` will match staging and development only
+* `master features/*` will match `master` and any branch that starts with `features/`
+* `*-test` will match any branch ending with `-test`, such as `rails-update-test`
+* `stages/* !stages/production` will match any branch starting with `stages/` except `stages/production`, such as `stages/demo`
+* `v*.0` will match any branch that begins with a `v` and ends with a `.0`, such as `v1.0`
+* `v* !v1.*` will match any branch that begins with a `v` unless it also begins with `v1.`, such as `v2.3`, but not `v1.1`
```

Comment:
So the change is basically that your disallowed branch can also include one that you've previously wildcarded? Is it worth saying that in an abstract way along with the examples? 

And that it doesn't work the other way round?

Reply:
Yeah, perhaps we could mention in the explanatory text that they can be combined. Check out the changelog for some possible words.

Reply:
How about that? I too can copy paste. :-)